.travis.yml

```diff
@@ -4,7 +4,16 @@ install:
 - pip install --user mkdocs mkdocs-material
 script:
 - mvn test -B
-- if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then cd documentation && mkdocs build --clean --verbose --strict && cd ..; fi;
+- |
```

Comment:
I presume this lets you spread the if then else over a few lines. Is that correct?

Reply:
yes